chcemvediet/tests/__init__.py

```diff
@@ -20,3 +29,149 @@ def setup_test_environment(self, **kwargs):
+        return self._call_with_defaults(District.objects.create, kwargs, {
+                u'id': name,
+                u'name': name,
+                u'region': self.region,
+                })
+
+    def _create_municipality(self, **kwargs):
+        name = u'SK{:05d}'.format(self.counter.next())
+        return self._call_with_defaults(Municipality.objects.create, kwargs, {
+                u'id': name,
+                u'name': name,
+                u'region': self.region,
+                u'district': self.district,
+                })
+
+    def _create_neighbourhood(self, **kwargs):
+        name = u'SK{:05d}'.format(self.counter.next())
+        return self._call_with_defaults(Neighbourhood.objects.create, kwargs, {
+                u'id': name,
+                u'name': name,
+                u'region': self.region,
+                u'district': self.district,
+                u'municipality': self.municipality,
+                })
+
+    def _create_obligee(self, **kwargs):
+        return self._call_with_defaults(Obligee.objects.create, kwargs, {
+                u'official_name': u'Default Testing Official Name',
+                u'name': u'Default Testing Name {:5d}'.format(self.counter.next()),
+                u'name_genitive': u'Default Testing Name genitive',
+                u'name_dative': u'Default Testing Name dative',
+                u'name_accusative': u'Default Testing Name accusative',
+                u'name_locative': u'Default Testing Name locative',
+                u'name_instrumental': u'Default Testing Name instrumental',
+                u'gender': Obligee.GENDERS.MASCULINE,
+                u'ico': u'00000000',
+                u'street': u'Default Testing Street',
+                u'city': u'Default Testing City',
+                u'zip': u'00000',
+                u'iczsj': self.neighbourhood,
+                u'emails': u'default_testing_mail@example.com',
+                u'latitude': 0.0,
+                u'longitude': 0.0,
+                u'type': Obligee.TYPES.SECTION_1,
+                u'official_description': u'Default testing official description.',
+                u'simple_description': u'Default testing simple description.',
+                u'status': Obligee.STATUSES.PENDING,
+                u'notes': u'Default testing notes.',
+                })
+
+    def _create_obligeetag(self, **kwargs):
+        return self._call_with_defaults(ObligeeTag.objects.create, kwargs, {
+                u'key': u'Default Testing Key',
+                u'name': u'Default Testing Name',
+                })
+
+    def _create_obligeegroup(self, **kwargs):
+        return self._call_with_defaults(ObligeeGroup.objects.create, kwargs, {
+                u'key': u'Default Testing Key',
+                u'name': u'Default Testing Name',
+                u'description': u'Default Testing Description',
+                })
+
+    def _create_obligeealias(self, **kwargs):
```

Comment:
@viliambalaz Este preco `obligeealias` volas takto spojene a nie `obligee_alias`? Podobne ako napriklad `inforequest_draft` mame s podciarkovnikom. Spolu to mame iba na miestach, kde si to vynucuje django. Napriklad `obligeealias_set` je spolu tak ako aj `inforequestdraft_set`. V nazvoch funkcii a lokalnych premennych to vsak mame vsade s `_`, kedze su to dva slova.

Rovnako `obligeegroup` a `obligeetag`

Reply:
:ok:

Reply:
@viliambalaz Pls aj nazvy premennych davaj `obligee_tag` alebo skratene `tag`, nie `obligeetag`. Rovnako u ostatnych.

Pouzivame "snake_case" a "obligee" a "tag" su dva slova.

Reply:
:ok: